packages/backend/src/connector/core/test/middleware/account-middleware.test.ts

```diff
@@ -97,6 +97,35 @@ describe('Account Middleware', () => {
     expect(ctx.accounts.outgoing).toEqual(outgoingAccount)
   })
 
+  test('create TB account for PENDING incoming payment', async () => {
```

Comment:
You could make this a [`test.each`](url) to also test that the middleware handles `createLiquidityAccount` throwing. (See: [mockRejectedValueOnce](https://jestjs.io/docs/mock-function-api#mockfnmockrejectedvalueoncevalue))